src/meta/src/hummock/compaction/compaction_config.rs

```diff
@@ -66,6 +66,7 @@ impl CompactionConfigBuilder {
                 max_sub_compaction: DEFAULT_MAX_SUB_COMPACTION,
                 max_space_reclaim_bytes: DEFAULT_MAX_SPACE_RECLAIM_BYTES,
                 split_by_state_table: false,
+                split_weight_by_vnode: 0,
```

Comment:
How will `split_weight_by_vnode` be calculated in this pr? I don't seem to find the relevant logic.

Reply:
> How will `split_weight_by_vnode` be calculated in this pr? I don't seem to find the relevant logic.

will always be 0